src/lib/ui/ClimateSettingForm/ClimateSettingScheduleForm.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames'
+import { useState } from 'react'
+import { Controller, useForm } from 'react-hook-form'
+import type { ClimateSetting } from 'seamapi'
+
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+
+import { ClimateSettingScheduleDeviceSelect } from './ClimateSettingScheduleDeviceSelect.js'
```

Comment:
```suggestion
import { ClimateSettingScheduleDeviceSelect } from 'src/lib/ui/ClimateSettingForm/ClimateSettingScheduleDeviceSelect.tsx'
```

Reply:
I'll see about making vscode do this for me

Reply:
This gives: `An import path can only end with a '.tsx' extension when 'allowImportingTsExtensions' is enabled.ts(5097)`

So I can follow up with a PR that allows this, it's a bit more natural